src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -72,7 +72,7 @@ private void processBlob(String blobName, String containerName) {
                 .getBlobContainerClient(containerName)
                 .getBlobClient(blobName);
 
-            Instant blobCreationDate = blobClient.getProperties().getCreationTime().toInstant();
+            Instant blobCreationDate = blobClient.getProperties().getLastModified().toInstant();
```

Comment:
Creation date is null. This might be Azurite bug, not sure.
We use 'last modified' date in bulk-scan-processor as well.

Reply:
good catch. good hypothesis too